rdr_service/resource/generators/code.py

```diff
@@ -29,7 +28,6 @@ def make_resource(self, _pk):
             return generators.ResourceRecordSet(schemas.CodeSchema, data)
 
 
-@task_auth_required
```

Comment:
Is this an endpoint? If so, is it protected another way?

Reply:
This is showing the line is being removed from the file.

Reply:
Right, sorry I wasn't very clear. I was referring to the `rebuild_codebook_resources_task` method below as the potential endpoint. By removing the task_auth_required decorator, does that expose it a potentially unwanted way?

Reply:
If you follow the usages for the `rebuild_codebook_resources_task` function you'll see that those functions have either the task API or cron API decorators.  Really, I think I copied that decorator there and accidentally let it slip into a commit.

Reply:
Makes sense 👍